src/layers/conv.jl

```diff
@@ -127,24 +127,26 @@ function (l::GCNConv)(g::AbstractGNNGraph,
     T = eltype(xi)
 
     if g isa GNNHeteroGraph
-        d = degree(g, g.etypes[1], T; dir = :in)
+        din = degree(g, g.etypes[1], T; dir = :in)
+        dout = degree(g, g.etypes[1], T; dir = :out)
     else
         if edge_weight !== nothing
-            d = degree(g, T; dir = :in, edge_weight)
+            din = dout = degree(g, T; dir = :in, edge_weight)
         else
-            d = degree(g, T; dir = :in, edge_weight = l.use_edge_weight)
+            din = dout = degree(g, T; dir = :in, edge_weight = l.use_edge_weight)
         end
     end
-    c = norm_fn(d)
-    !(g isa GNNHeteroGraph) ? xj = xj .* c' : Nothing
+    cout = norm_fn(dout)
+    cin = norm_fn(din)
```

Comment:
we don't want to call norm_fn twice if not needed, so it's better to define `cin` and `cout` in the branches above